### PR TITLE
Configure relationships

### PR DESCRIPTION
**Pull request by kristupas-g**

### CODE REVIEW

File: PoSSapi/src/Domain/Entities/Business.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+namespace PoSSapi.Domain.Entities;
+
+public class Business
+{
+    public Guid Id { get; set; }
+    public ICollection<BusinessLocation> Location { get; set; }
+    //Needs BusinessType enum but I am unsure what values it could have
```
Comment (baikauskaite) [inline PoSSapi/src/Domain/Entities/Business.cs:7]: `ManagerId` should be here
image

Comment (baikauskaite) [reply PoSSapi/src/Domain/Entities/Business.cs:7]: Maybe business types: Restaurant, Service? Would be good enough

File: PoSSapi/src/Domain/Entities/BusinessLocation.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+namespace PoSSapi.Domain.Entities;
+
+public class BusinessLocation
+{
+    public Guid Id { get; set; }
+    public string Location { get; set; }
```
Comment (baikauskaite) [inline PoSSapi/src/Domain/Entities/BusinessLocation.cs:6]: Shouldn't `BusinessLocation` have these fields?
image

File: PoSSapi/src/Domain/Entities/User.cs [hunk 3]
```diff
@@ -5,12 +5,17 @@ namespace PoSSapi.Domain.Entities;
...
     public int Age { get; set; }
     public string Username { get; set; }
     public string Email { get; set; }
     public string Password { get; set; }
     public UserType UserType { get; set; }
+    public string Position { get; set; }
+    public decimal Salary { get; set; }
+    public ICollection<Order> Orders { get; set; }
+    public ICollection<Payment> Payments { get; set; }
+    public ICollection<Reservation> Reservations { get; set; }
```
Comment (baikauskaite) [inline PoSSapi/src/Domain/Entities/User.cs:20]: image

File: PoSSapi/src/Domain/Entities/Business.cs (hunk 1 above)
Comment (Arnasst) [inline PoSSapi/src/Domain/Entities/Business.cs:7]: Business type was not mentioned in the QuickFix design document, only in the schemas (and not in any controller :D) (other enum values were mentioned in the doc). I'd quess this could be left out, since it won't be available through the API and the given design does not seem to support services.

**Review by baikauskaite — COMMENTED** [review]
Nice job cleaning up :star2: :star2: I looked at the swagger of QuickFix and left some comments about things I noticed

File: PoSSapi/src/Domain/Entities/User.cs [hunk 4]
```diff
@@ -5,12 +5,17 @@ namespace PoSSapi.Domain.Entities;
...
+    public Business Business { get; set; }
     public string Name { get; set; }
     public string Surname { get; set; }
     public int Age { get; set; }
     public string Username { get; set; }
     public string Email { get; set; }
     public string Password { get; set; }
     public UserType UserType { get; set; }
+    public string Position { get; set; }
+    public decimal Salary { get; set; }
```
Comment (Arnasst) [inline PoSSapi/src/Domain/Entities/User.cs:17]: These seem to better belong to StaffMember

File: PoSSapi/src/Domain/Entities/User.cs (hunk 3 above)
Comment (Arnasst) [inline PoSSapi/src/Domain/Entities/User.cs:20]: Why is this needed here, doesn't it provide duplicate information that could be gotten in another way?

File: PoSSapi/src/Application/Common/Interfaces/IApplicationDbContext.cs [hunk 5]
```diff
@@ -5,9 +5,16 @@ namespace PoSSapi.Application.Common.Interfaces;
 
 public interface IApplicationDbContext
 {
-    DbSet<TodoList> TodoLists { get; }
-
-    DbSet<TodoItem> TodoItems { get; }
+    DbSet<Business> Businesses { get; set; }
+    DbSet<BusinessLocation> BusinesseLocations { get; set; }
```
Comment (Arnasst) [inline PoSSapi/src/Application/Common/Interfaces/IApplicationDbContext.cs:9]: Typo `BusinessLocations`

**Review by baikauskaite — APPROVED** [review]
Great job fixing :partying_face: :partying_face:
